src/views/minor/NavBar.vue

```diff
@@ -1,105 +1,142 @@
                 </RouterLink>
                 <button
-                    class="navbar-toggler"
+                    id="navbar-toggler"
                     type="button"
-                    data-bs-toggle="collapse"
-                    data-bs-target="#navbarSupportedContent"
                     aria-controls="navbarSupportedContent"
-                    aria-expanded="false"
-                    aria-label="Toggle navigation.">
-                    <span class="navbar-toggler-icon"></span>
+                    :aria-expanded="menuOpen"
+                    aria-label="Toggle navigation."
+                    v-on:click="toggleMenu">
+                    <span class="material-icons-sharp">menu</span>
                 </button>
-                <div
-                    class="collapse navbar-collapse"
-                    id="navbarSupportedContent">
-                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                        <li class="nav-item">
-                            <RouterLink class="nav-link" to="/scope">
-                                The 'Scope
-                            </RouterLink>
-                        </li>
-                        <li class="nav-item">
-                            <a
-                                href="/doc/doc/about/index.html"
-                                class="nav-link">
-                                About
-                            </a>
-                        </li>
-                        <li class="nav-item">
-                            <a href="/doc/" class="nav-link">
-                                Documentation
-                            </a>
-                        </li>
-                    </ul>
-                    <span class="navbar-text">
-                        {{ copyright }}
-                        <br />
-                        Licensed Under the
-                        <a href="https://opensource.org/licenses/MIT">
-                            MIT License</a
-                        >
-                    </span>
-                    <span class="navbar-text"> </span>
-                </div>
+            </div>
+            <div id="navbar-links" :class="{open: menuOpen}">
+                <RouterLink
+                    class="nav-link"
+                    to="/gallery"
+                    v-on:click="closeMenu">
+                    Gallery
+                </RouterLink>
+
+                <a href="/doc/doc/about/index.html" class="nav-link">
+                    About
+                </a>
+
+                <a href="/doc/" class="nav-link"> Documentation </a>
             </div>
         </nav>
     </header>
 </template>
 
 <script lang="ts">
-    const currentYear = new Date().getFullYear()
-    const copyrightText =
-        `Copyright © 2020-${currentYear}`
-        + ' Regents of the University of Colorado'
-    export default {
-        data() {
+    import {defineComponent} from 'vue'
+
```

Comment:
Umm for us to ever publish this it will need the copyright notice somewhere. Like with the acknowledgments link, we're not going to require that for this PR but we do want a definite design of where the copyright notice will go. Thanks!

Reply:
Okay, we'll discuss this next week!

Reply:
I see there is a copyright notice on the figma, so resolving under the presumption that it will be handled in a future PR to bring us to feature completion.